easybuild/easyconfigs/m/matplotlib/matplotlib-2.0.2-foss-2017a-Python-2.7.13.eb

```diff
@@ -0,0 +1,44 @@
+easyblock = 'Bundle'
+
+name = 'matplotlib'
+version = '2.0.2'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://matplotlib.org'
+description = """matplotlib is a python 2D plotting library which produces publication quality figures in a variety of
+ hardcopy formats and interactive environments across platforms. matplotlib can be used in python scripts, the python
+ and ipython shell, web application servers, and six graphical user interface toolkits."""
+
+toolchain = {'name': 'foss', 'version': '2017a'}
+
+# this is a bundle of Python packages
+exts_defaultclass = 'PythonPackage'
+
+dependencies = [
+    ('Python', '2.7.13'),
+    ('freetype', '2.7.1'),
```

Comment:
@jhein32 the easyconfig for `freetype` 2.7.1 included in `develop` uses `libpng` 1.6.28, hence the version conflict you are seeing

We have a separate `freetype` 2.7.1 easyconfig that uses libpng 1.6.29 though, you can instruct EasyBuild to pick that up instead as follows:

```python
('freetype', '2.7.1', '-libpng-1.6.29'),
```

To avoid duplicating the `libpng` version we usually do something like:

```python
libpng_ver = '1.6.29'
dependencies = [
    ...
    ('libpng', libpng_ver),
    ('freetype', '2.7.1', '-libpng'%s' % libpng_ver),
```